### PR TITLE
nolo tlc ciel 12_3 to 12_3b

### PR DESCRIPTION
**Pull request by nolo16**
babe...... wake up........ back to ciel route................
image

### CODE REVIEW

File: script/Ciel/Day 12/12_01_CIEL12_3.txt [hunk 1]
```diff
@@ -1,17 +1,17 @@
 [sha:f83a7acadb446820f9389112804983a7f328e29c]{
 -- Page 10008, Offset 31042.
 -- 「あー……でも先輩、茶道部については疑問があるというか、質問があるというか。
-"Ah... but Senpai, I have a question regarding the tea ceremony club, or maybe it's more like an enquiry.
+"Ah... but Senpai, there's something regarding the tea ceremony club that's been weighing on my mind. I guess you could call it an inquiry of sorts.
```
Comment (Avuxo) [inline script/Ciel/Day 12/12_01_CIEL12_3.txt:4]: tl seems pretty overwritten, here's my TL:
> Ah... but Senpai, there's something I guess I don't really get about the tea ceremony club. I guess you could say I have a question.

File: script/Ciel/Day 12/12_01_CIEL12_3.txt [hunk 2]
```diff
@@ -1,17 +1,17 @@
...
 -- Page 10008, Offset 31043.
 -- 前に部員はいないって言ってましたけど、この学校、初めから茶道部そのものがないんじゃないですか？」
-You said before that there weren't any other members, but wasn't the tea ceremony club not around from the start?"
+You said before that there weren't any other members, but did the tea ceremony club even exist in the first place?"
 }
 [sha:aa5ebec0d20c3793b2a88f207fc2ab9ba24dbb01]{
 -- Page 71, Offset 31044.
 -- 「……あの。もしかして本当になかった？
-"...Um. Could it be that it really wasn't?
+"...Um. Could it be that it really didn't?
```
Comment (Avuxo) [inline script/Ciel/Day 12/12_01_CIEL12_3.txt:14]: Don't tell me it really didn't?

File: script/Ciel/Day 12/12_01_CIEL12_3.txt [hunk 3]
```diff
@@ -76,22 +76,22 @@ A little less than a minute of complete silence.
...
 -- Page 75, Offset 31057.
 -- はあ、と根負けしたように先輩はため息をつく。
-Senpai breathes out a sigh as if to admit defeat.
+Senpai breathes a sigh as if admitting defeat.
 }
 [sha:5cf49c39de3fe207399ee480ce977da75587474f]{
 -- Page 76, Offset 31058.
 -- 「言っておきますが、暗示というものはそんなに都合のいいものじゃないんですよ。人を言いなりになんかできませんし、無理難題を通すコトもできません。
-"Just so you know, suggestion is not as convenient as you think it is. I can't make people do exactly as they're told, and I can't make unreasonable demands.
+"Just so you know, suggestion isn't as convenient as you think it is. I can't make people do exactly as they're told, and I can't make unreasonable demands either.
```
Comment (Avuxo) [inline script/Ciel/Day 12/12_01_CIEL12_3.txt:84]: should references to 暗示 be capitalized or is this literally just her using the "industry term" for hypnotism

File: script/Ciel/Day 12/12_01_CIEL12_3.txt [hunk 4]
```diff
@@ -76,22 +76,22 @@ A little less than a minute of complete silence.
...
 }
 [sha:ff93167f4e3990973fd836e88c475fabea4a8a36]{
 -- Page 76, Offset 31059.
 -- 暗示とは物事の捉え方を変える、のではなく逸らす、というのが大前提なんです」
-The main principle behind suggestion is not to change how one perceive things, but to make one avoid them."
+The main premise behind suggestion isn't changing one's perception, but rather diverting it."
```
Comment (Avuxo) [inline script/Ciel/Day 12/12_01_CIEL12_3.txt:89]: isn't to change one's perspective, but rather to divert it

File: script/Ciel/Day 12/12_01_CIEL12_3.txt [hunk 5]
```diff
@@ -121,32 +121,32 @@ For example, instead of 'I love it, so I'll eat it', I can suggest something lik
...
 -- Page 80, Offset 31066.
 -- ……なんだ、やっぱりなんでも出来るじゃないですか、それ」
-...Well, then it's like you can make them do pretty much anything."
+...Well, that means you can pretty much do anything with it, after all."
 }
 [sha:e5624e5d04f28340c1ff5bd39085f59a88fc2d69]{
 -- Page 81, Offset 31067.
 -- 「だから何でもじゃありませんってば。
-"Like I said, it's not like I have complete control.
+"As I said, I can't do %{i}anything%{/i}.
```
Comment (Avuxo) [inline script/Ciel/Day 12/12_01_CIEL12_3.txt:129]: italics should be on "do", not "anything"

File: script/Ciel/Day 12/12_01_CIEL12_3.txt [hunk 6]
```diff
@@ -261,12 +261,12 @@ Hold on, give me a sec...!"
...
 -- Page 90, Offset 31094.
 -- はやる気持ちを抑えられず、あたふたと携帯を操作する。
-Unable to control my excited self, I hastily fumble around on my phone.
+Unable to control my excitement, I hastily fumble around with my phone.
 }
 [sha:eb0bcc884f4a0b942d520ddf7b65e1cb1f6b111e]{
 -- Page 90, Offset 31095.
 -- ……我ながら情けない。生まれつき小市民なせいか、突然の幸福にまったく対応できていない。
-...A pathetic display if I say so myself. Perhaps it's because I was born in the lower middle class, but I can't deal with this sudden influx of happiness whatsoever.
+...A pathetic display, if I do say so myself. Perhaps it's because I'm a born member of the lower middle class, but I have absolutely no idea how to deal with this sudden influx of happiness.
```
Comment (Avuxo) [inline script/Ciel/Day 12/12_01_CIEL12_3.txt:269]: i cannot even begin to understand this logic

File: script/Ciel/Day 12/12_01_CIEL12_3B.txt [hunk 7]
```diff
@@ -21,22 +21,22 @@ As soon as fourth period ended, Kamokashigeru-sensei made the following announce
 [sha:47e571b817801d34d6632e8fff69c33316054ef3]{
 -- Page 94, Offset 31106.
 -- 今日の授業は私でおしまいです。校舎に欠陥が発見されたので、午後は工事のための調査が入ります。
-There will be no more other classes today. There was a defect found in the school building, so a survey will be carried out this afternoon for construction.
+This will be your last class for the day. A defect was found in the school building, so an inspection for construction will be carried out this afternoon.
```
Comment (Avuxo) [inline script/Ciel/Day 12/12_01_CIEL12_3B.txt:24]: "repairs" probably makes more sense than "construction"

File: script/Ciel/Day 12/12_01_CIEL12_3B.txt [hunk 8]
```diff
@@ -21,22 +21,22 @@ As soon as fourth period ended, Kamokashigeru-sensei made the following announce
...
 }
 [sha:06b656acba4348a0942d35a88d5ac94af9cd8a50]{
 -- Page 94, Offset 31107.
 -- 皆さんはこのまま午後２時までに下校するように。食堂は１時まで開いていますので、定期食券をもっている生徒の方は利用してもかまいません。ではでは』
-As such, please make your way home before 2:00 PM. The cafeteria will be open until 1:00, so it's okay for students with fixed meal tickets to go ahead and use them. Now if you excuse me."
+As such, please make your way home before 2:00 PM. The cafeteria will be open until 1:00, so students with fixed meal tickets may go ahead and use them. Now, if you excuse me."
```
Comment (Avuxo) [inline script/Ciel/Day 12/12_01_CIEL12_3B.txt:29]: if you'll excuse me

File: script/Ciel/Day 12/12_01_CIEL12_3B.txt [hunk 9]
```diff
@@ -21,22 +21,22 @@ As soon as fourth period ended, Kamokashigeru-sensei made the following announce
...
 }
 [sha:33d46e236993b49cde6d0d7b273970181baf24c8]{
 -- Page 95, Offset 31108.
 -- 発見された欠陥は地盤沈下のたぐいらしく、ロアの柩があった地下空洞との関連性が気になったが、どうあれ吸血鬼はもういない。
-The defect that was found seemed to match that of a land subsidence, and I wondered whether it's related to the underground cave where Roa's coffin was, but in any case, the vampire is no longer around.
+The defect that was found seemed to be some sort of land subsidence, so I wondered if it had something to do with the underground cavity Roa's coffin was in, but regardless, the vampire is no longer around.
```
Comment (Avuxo) [inline script/Ciel/Day 12/12_01_CIEL12_3B.txt:34]: was in. Regardless, that vampire is no longer around.

File: script/Ciel/Day 12/12_01_CIEL12_3B.txt [hunk 10]
```diff
@@ -21,22 +21,22 @@ As soon as fourth period ended, Kamokashigeru-sensei made the following announce
...
 }
 [sha:1251b6c4c67d3b1365083a3ca4e168bb195f5414]{
 -- Page 95, Offset 31109.
 -- そもそもあんな物騒な場所と繋がっていたら先輩が黙っていないだろうし、俺たちには関係のない話と思われる。
-And besides, if it was really related to such a dangerous place, then Senpai wouldn't have kept quiet about it, and it seems like it has nothing to do with us.
+Besides, if it was related to such a dangerous place then Senpai would've told me about it, so I figure it has nothing to do with us.
```
Comment (Avuxo) [inline script/Ciel/Day 12/12_01_CIEL12_3B.txt:39]: place, then Senpai

File: script/Ciel/Day 12/12_01_CIEL12_3B.txt [hunk 11]
```diff
@@ -56,22 +56,22 @@ Should I wait for Senpai to come, or should I go to the tea ceremony room?
...
 -- Page 97, Offset 31113.
 -- いや、いっそさっき手に入れたばかりのメアドを有効活用するべきか……と。
-No, maybe I should make use of her mail address that I just received...
+No, maybe I should put the mail address that I just received to use...
 }
 [sha:418ff0c5156a1d0802e0527fa7975442463df6a1]{
 -- Page 97, Offset 31114.
 -- 午前中ひたすら机につっぷしていた三年寝太郎が、怪しい笑みをうかべてこちらに近寄ってきた。
-The sleeping legend, Sannen Netaro, who had spent the entire morning face-down on his desk, approaches me with a suspicious-looking smile.
+The sleeping legend, Sannen Netaro, who had spent the entire morning face-down on his desk, approaches me with a suspicious smile.
```
Comment (Avuxo) [inline script/Ciel/Day 12/12_01_CIEL12_3B.txt:64]: "Sannen Netaro" makes no sense in english (and it's improperly romanized). Can't think of any good english puns right now so maybe mark it / bring it up in the server

I do wonder, why 三年?

File: script/Ciel/Day 12/12_01_CIEL12_3B.txt [hunk 12]
```diff
@@ -196,17 +196,17 @@ The perpetrator is none other than Arihiko.
 [sha:5b77dc1351821c03c388e256c257d28603444ad8]{
 -- Page 108, Offset 31141.
 -- 「ずっとテメエの目の前にいたっつーの。そしてあまりのタワゴトに覚醒したっつーの。
-"I was in front of you this whole time. And now my eyes have been opened up to your bullshit.
+"I was in front of you this whole time. And now my eyes have been opened up because of all your bullshit.
```
Comment (Avuxo) [inline script/Ciel/Day 12/12_01_CIEL12_3B.txt:199]: drop "and"

File: script/Ciel/Day 12/12_01_CIEL12_3B.txt [hunk 13]
```diff
@@ -221,12 +221,12 @@ Reflexively, I exchange glances with Ciel-senpai.
...
 -- Page 110, Offset 31146.
 -- 「あのね、キミたち。せっかくの半ドンをどうしてそういう風にしか使えねえんだよ。茶道室に寄っていくとか正気を失う、いや疑うレベルだからな。
-"Come on now, you two. We finally get half the day off, so why is that the only thing you guys think of doing? Dropping by the tea ceremony room is at the level of crazy, or unbelievable rather.
+"C'mon now, you two. We finally get half the day off, so why is that the only thing you guys think of doing? Dropping by the tea ceremony room is crazy, unbelieveable even.
 }
 [sha:eb3aef1673a9c77c2b4f83523f6b8e87a00c0b7e]{
 -- Page 110, Offset 31147.
 -- ここは人工失楽園、二十一世紀のトーキョーだぜ？　もっとマシな遊び場はあそこに山ほど転がってるだろうが！」
-Aren't we living in the man-made Paradise Lost otherwise known as 21st century Tokyo? I'm sure there are plenty of better places to play around than there!"
+Aren't we living in the man-made Paradise Lost otherwise known as 21st century Tokyo? I'm sure there are lots of better places to hang out than there!"
```
Comment (Avuxo) [inline script/Ciel/Day 12/12_01_CIEL12_3B.txt:229]: We're living in the man-made Paradise Lost that is twenty-first century Tokyo!

File: script/Ciel/Day 12/12_01_CIEL12_3B.txt [hunk 14]
```diff
@@ -306,12 +306,12 @@ I steal a glance at Ciel-senpai.
 [sha:b11ede70be08e4c2b71c4af33393b7346ad8911f]{
 -- Page 115, Offset 31163.
 -- この人は街で派手に遊ぶより、公園をのんびり散策したりする方が性に合っているような気がして、どうも誘えない。
-It's hard to ask her out since this person seems more suited to taking leisurely strolls at the park than playing around extravagantly at the city.
+I can't ask her out. After all, she seems more suited to taking a leisurely stroll at the park than lavishly playing around in the city.
```
Comment (Avuxo) [inline script/Ciel/Day 12/12_01_CIEL12_3B.txt:309]: she seems more suited to a leisurely stroll in the park than she is playing around in the city

**Review by Avuxo — APPROVED** [review]
glad to have you back :arcsmile: otsukare

File: script/Ciel/Day 12/12_01_CIEL12_3.txt (hunk 3 above)
Comment (nolo16) [reply script/Ciel/Day 12/12_01_CIEL12_3.txt:84]: figure its the latter

File: script/Ciel/Day 12/12_01_CIEL12_3.txt (hunk 6 above)
Comment (nolo16) [reply script/Ciel/Day 12/12_01_CIEL12_3.txt:269]: what-did-he-mean-by-this-dove

File: script/Ciel/Day 12/12_01_CIEL12_3B.txt (hunk 11 above)
Comment (nolo16) [reply script/Ciel/Day 12/12_01_CIEL12_3B.txt:64]: marked

**Pull request merged**